index.js

```diff
@@ -94,6 +94,7 @@ function fastifyMultipart (fastify, options, done) {
             const key = reqBodyKeys[i]
             const field = req.body[key]
 
+            /* istanbul ignore else */
```

Comment:
Maybe justify why it's being ignored for posterity's sake?
i.e. `/*istanbul ignore else: cba to test it */`